plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchConnector.java

```diff
@@ -53,6 +59,12 @@ public ElasticsearchConnector(
         this.splitManager = requireNonNull(splitManager, "splitManager is null");
         this.pageSourceProvider = requireNonNull(pageSourceProvider, "pageSourceProvider is null");
         this.nodesSystemTable = requireNonNull(nodesSystemTable, "nodesSystemTable is null");
+        this.sessionProperties = ImmutableList.of(
+                PropertyMetadata.booleanProperty(
+                        AGGREGATION_PUSHDOWN_ENABLED,
+                        "Enable aggregation pushdown",
+                        true,
```

Comment:
Please define a config property as well, so that it can be enabled or disabled for all users.

Reply:
@findepi could you please show me an example?

Reply:
sure, see `io.trino.plugin.jdbc.JdbcMetadataConfig#aggregationPushdownEnabled` and also associated getters and setters.
the configs are driven by annotations